crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -531,9 +531,9 @@ en:
       status: 'Status'
       available: 'Available'
       unavailable: 'Not present'
-      md5_check: 'MD5 Check'
-      md5_ok: 'OK'
-      md5_failed: 'Failed'
+      repo_check: 'Repository Check'
```

Comment:
What about just `Check`? It's obvious that it's repo related as you are anyway in the repositories list.